render/detailed/links.go

```diff
@@ -0,0 +1,169 @@
+package detailed
+
+import (
+	"bytes"
+	"net/url"
+	"strings"
+	"text/template"
+
+	"github.com/weaveworks/scope/probe/docker"
+	"github.com/weaveworks/scope/report"
+
+	"github.com/ugorji/go/codec"
+)
+
+// MetricLink describes a link referencing a metric.
+type MetricLink struct {
+	// References the metric id
+	ID       string `json:"id,omitempty"`
+	Label    string `json:"label"`
+	URL      string `json:"url"`
+	Priority int    `json:"priority"`
+}
+
+// Variable name for the query within the metrics graph url
+const urlQueryVarName = ":query"
+
+var (
+	// As configured by the user
+	metricsGraphURL = ""
+
+	// Available metric links
+	linkTemplates = []MetricLink{
+		{ID: docker.CPUTotalUsage, Label: "CPU", Priority: 1},
+		{ID: docker.MemoryUsage, Label: "Memory", Priority: 2},
+	}
+
+	// Prometheus queries for topologies
+	topologyQueries = map[string]map[string]*template.Template{
+		report.Pod: {
+			docker.MemoryUsage:   prepareTemplate(`sum(container_memory_usage_bytes{pod_name="{{.Label}}"})`),
```

Comment:
I think there should be some explanation here about where these rules come from: are they assuming some particular exporter is in place?  Are they assuming anything in forwarding or recording rules?